### ISSUE TITLE
objectives, outcomes, standards layout and accessibility in eBook

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#892]
Objectives (implemented), outcomes (not), and standards (not) are each more like metadata for a division more so than content (imho). As mentioned in #890, perhaps these should not appear in the eBook as expanded content at the beginning of a division.

For one thing, their presence before an introduction would mean that the introduction must have a heading for the sake of accessibility. That is undesirable for non-titled introductions, and would probably bring about more use of the CSS that puts the headings off screen. (Which is current best practice, but still feels like a hack.)

But more fundamentally, if they are metadata, maybe they should appear as something like an aside. Or something that can be expanded from a knowl. In the DOM, for accessibility they may need to come as h2's prior to the section title's h1. 

Unclear what this would look like for objectives/outcomes/standards associated with, say, a subsection when you chunk by section.

**Comment by davidfarmer** [comment]
Independently I was planning to suggest that objetives in HTML be
marked up as an "aside".

If we do that, do we still need to wrap an HTML "section" around the
introdution to the PTX section?

That is, the page might be:

section
   h1
   aside
   p
   p

   section
   h2
   etc

**Comment by Alex-Jordan** [comment]
Will the aside itself have a heading? The problem is sort of this: if I navigate to the heading that says "Objectives" and I think that I don't need to read that, I will skip to the next heading. In this case, the h2 in your outline. And I skip over the two p's from the introduction. I am wondering if the outline could be like this:

```
aside (objectives for the PTX section)
  h2
section 
  h1
  p
  p

  aside (objectives for the PTX subsection)
    h2
  section
  h2
  etc
```

The potential issue there is a reader who goes straight to the h1 without being aware of earlier content. But I did see examples in a tutorial on headings that said it was OK to have metadata-ish stuff in headings earlier than the h1.

**Comment by rbeezer** [comment]
The aria roles have a "main" region, which we might call "main content".  Then 
they have "complementary" region.  I'll guess you can have multiple 
complementary regions so long as they are labeled.  It might be easy to split 
out an early one, harder to split out a later one.  Perhaps then the heading 
level is not as critical?

**Comment by Alex-Jordan** [comment]
Yes, perhaps David's outline will work, even when the aside has a heading, if aria identifies the aside as complementary. Actually "complementary" is redundant for an aside. The html-aria spec](https://www.w3.org/TR/html-aria/) says so. We should try David's outline then until we can build with it and have an evaluation.
